### PR TITLE
[onert/python] Introduces loss API

### PR DESCRIPTION
**Pull request by ragmani**
This commit introduces loss API.
  - cce.py : Categorical CrossEntropy Loss
  - mse.py : Mean Squared Error Loss

ONE-DCO-1.0-Signed-off-by: ragmani <ragmani0216@gmail.com>

### CODE REVIEW

**Comment by ragmani** [comment]
For https://github.com/Samsung/ONE/issues/14505
Draft : https://github.com/Samsung/ONE/pull/14492

File: runtime/onert/api/python/package/experimental/train/losses/cce.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+            float or np.ndarray: Computed loss value(s).
+        """
+        epsilon = 1e-7  # Prevent log(0)
+        y_pred = np.clip(y_pred, epsilon, 1 - epsilon)
+        loss = -np.sum(y_true * np.log(y_pred), axis=1)
+
+        if self.reduction == "mean":
+            return np.mean(loss)
+        elif self.reduction == "sum":
+            return np.sum(loss)
```
Comment (zetwhite) [inline runtime/onert/api/python/package/experimental/train/losses/cce.py:33]: May I ask why loss function is implemented here? 
I thought the loss function executed inside the ONERT here -  https://github.com/Samsung/ONE/blob/da3fdd292d288f22d3abd393845c74c18381ea76/runtime/onert/backend/train/ops/LossCategoricalCrossentropyLayer.h#L41-L42

Comment (ragmani) [reply runtime/onert/api/python/package/experimental/train/losses/cce.py:33]: Right, the loss function run inside there. It seems that this function is unnecessary.

Comment (ragmani) [reply runtime/onert/api/python/package/experimental/train/losses/cce.py]: I removed it.

**Review by zetwhite — APPROVED** [review]
LGTM

**Review by hseok-oh — APPROVED** [review]
LGTM
